.github/workflows/sycl-detect-changes.yml

```diff
@@ -57,6 +57,8 @@ jobs:
             drivers:
               - devops/dependencies.json
               - devops/scripts/install_drivers.sh
+            devigccfg:
```

Comment:
I'm not sure if it should be
```
devigccfg: &devigccfg
  - devops/dependencies-igc-dev.json
drivers:
  - *devigccfg
  - devops/dependencies.json
  - devops/scripts/install_drivers.sh
```

instead, but either way is good enough for now.